components/x-teaser/src/Title.jsx

```diff
@@ -28,9 +28,8 @@ export default ({ title, altTitle, headlineTesting, relativeUrl, url, indicators
 			{indicators && indicators.accessLevel === 'premium' ? (
 				<span>
 					{' '}
-					<span className={premiumClass} aria-label="Premium content">
-						Premium
-					</span>
+					<span className={premiumClass}>Premium</span>
+					<span className="o-normalise-visually-hidden">Premium content</span>
```

Comment:
I belive they suggest to do this in the ticket. A screen reader should read it as a single phrase.
```suggestion
					<span className={premiumClass}>Premium</span>
					<span className="o-normalise-visually-hidden">&nbsp;content</span>
```

Reply:
+1 to this solution!

Reply:
thanks @umbobabo @apaleslimghost . I've just tried it with Voice Over and reads correctly "Premium content"